lnwallet/channel.go

```diff
@@ -1683,6 +1733,16 @@ func (lc *LightningChannel) SignNextCommitment() ([]byte, error) {
 	// latest commitment update.
 	lc.remoteCommitChain.addCommitment(newCommitView)
 
+	// If we are the channel initiator then we would have signed any sent fee
+	// update at this point, so mark this update as pending ACK, and set
+	// pendingFeeUpdate to nil. We can do this since we know we won't sign any
+	// new commitment before receiving a revoke_and_ack, because of the
+	// revocation window of 1.
+	if lc.channelState.IsInitiator {
+		lc.pendingAckFeeUpdate = lc.pendingFeeUpdate
```

Comment:
See comment above about the simplification which _should_ allow us to only track a single pending fee update variable.